README.adoc

```diff
@@ -16,7 +16,9 @@ MinGW-w64 port is supported in OCaml 5.0 and the Cygwin port is restored in 5.1.
 On Linux, native code support for RISC-V and s390x/IBM Z will be available in
 OCaml 5.1 and on Power in 5.2.
 
-On 32-bit systems, only the bytecode compiler will be supported.
+❗ Native compilation in OCaml 5 is only available on 64-bit targets.
+Native compilation on 32-bit systems is **not** available, nor are there plans to
```

Comment:
I think this wording is an improvement.  It could be made clearer still with temporal expressions that make explicit that there has been a change in OCaml 5, e.g. "From OCaml 5.0 onwards, native compilation is available only on [...]  is no longer available on [...]"

Reply:
Yes, I like this statement better too. I am forgetting the magic sequence in GitHub to "suggest edits".

Reply:
Amended as suggested. Thanks!